flow/connectors/eventhub/scoped_eventhub.go

```diff
@@ -9,25 +9,30 @@ import (
 // partition_column is the column in the table that is used to determine
 // the partition key for the eventhub. Partition value is one such value of that column.
 type ScopedEventhub struct {
-	PeerName           string
+	EventhubNamespace  string
 	Eventhub           string
 	PartitionKeyColumn string
 	PartitionKeyValue  string
 }
 
 func NewScopedEventhub(dstTableName string) (ScopedEventhub, error) {
-	// split by dot, the model is peername.eventhub.partition_key_column
+	// split by dot, the model is eventhub.eventhub_namespace.table_name.partition_key_column
 	parts := strings.Split(dstTableName, ".")
 
-	if len(parts) != 3 {
+	if len(parts) != 4 {
 		return ScopedEventhub{}, fmt.Errorf("invalid scoped eventhub '%s'", dstTableName)
 	}
 
-	// support eventhub name and partition key with hyphens etc.
-	eventhubPart := strings.Trim(parts[1], `"`)
-	partitionPart := strings.Trim(parts[2], `"`)
+	// support eventhub namespace, eventhub name, partition key with hyphens etc.
+	// part[2] will be some table identifier.
+	// It's just so that we have distinct destination table names
+	// in create mirror's table mapping.
+	// We can ignore it.
+	eventhubPart := strings.Trim(parts[0], `"`)
```

Comment:
These parts would ideally be sent as `[]string` in protocol, avoid this incorrect parsing